### DISCUSSION TITLE
popup boxes

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#69] [Q&A] [answered]
What are the types of popup boxes present in JavaScript?

**Comment by Bunny77K9** [comment, marked as answer]
There are three types of dialog boxes, which are used in JavaScript:

    Alert: Presents users with a message and an ‘Ok’ button
    Confirm: Gives the users a window with ‘Ok’ and ‘Cancel’ buttons
    Prompt: Shows the user input, alongside ‘Ok’ and ‘Cancel’ buttons
